CoreLayersView/nbproject/project.xml

```diff
@@ -103,6 +111,15 @@
                     </run-dependency>
                 </dependency>
             </module-dependencies>
+            <test-dependencies>
+                <test-type>
+                    <name>unit</name>
+                    <test-dependency>
+                        <code-name-base>org.netbeans.libs.testng</code-name-base>
```

Comment:
😄

Reply:
This is ok yes ? Just making sure it's just a laugh on behalf of tests 👍

Reply:
Yeah its a happy laugh to see tests - keep it up!